nuitka/plugins/PluginBase.py

```diff
@@ -910,3 +914,28 @@ def wrapped(*args, **kwargs):
                 return None
 
     return wrapped
+
+
+class TagContext(dict):
+    def __init__(self, full_name, *args, **kwargs):
+        self.full_name = full_name
+        super().__init__(*args, **kwargs)
+
+    def __getitem__(self, key):
+        try:
+            dict.__getitem__(self, key)
+
+        except KeyError:
+            if key.startswith("use_"):
+                Tracing.plugins_logger.info(
```

Comment:
One advantage of being in every plugin is that we can just `self.info` which will then give the plugin name.

Reply:
This isn't in a plugin.